py-tests/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM src-base:v1
+# In the future should look like basho/src-base
+
+MAINTAINER Korrigan Clark
+
+# This Dockerfile if built from korry8911/pyspark-riak-tester:v2 which has all dependencies
+# already downloaded and install. The only this this Dockerfile should do is build spark-connector repo and
+# run tests
+
+WORKDIR /root/pyspark-riak-tests
+
+# Add jar dependencies for pyspark-riak tests (jars located in the py-tests folder of kc-tests spark connector branch)
```

Comment:
We are supposed to test changes made to spark connector code and to python module every time. Therefor we cannot use pre-built jar and whl.